examples/upgrade_vars.yaml

```diff
@@ -1,4 +1,8 @@
 ---
+eus_intermediate_upgrade_version: ""  #OCP upgrade version eg. 4.11.2
+eus_upgrade_channel: ""               #OCP channel having required upgrade version available for cluster upgrade (stable-4.x, fast-4.x, candidate-4.x, eus-4.x) eg. stable-4.11
+eus_intermediate_upgrade_image: ""    #OCP upgrade image e.g. "quay.io/openshift-release-dev/ocp-release-nightly@sha256:xxxxx"
+upstream: ""                          #Set the URL for OCP update server eg. https://ppc64le.ocp.releases.ci.openshift.org/graph
```

Comment:
Any specific reason for adding `intermediate` in the variable names above and not in `eus_upgrade_channel`?
Also `upstream` is just applicable for eus upgrades right? if yes can we prefix it with `eus_`?

Reply:
Can we make it a structure of variables for EUS? Looking at this it is very difficult to understand without proper documentation.

Reply:
Regarding the `eus_upgrade_channel`, if it is set to eus channel (eus-4.10) then cluster will be upgraded from 4.8->4.9->4.10 through the same channel. 
ok I will prefix `eus_` for `upstream` because it used in eus upgrade file.

Reply:
Ok I will modify all the eus variables like `eus_<name>`, will remove `intermediate`